### ISSUE TITLE
j.clients.ssh.loadSSHAgent() doesn't load id_rsa

### ISSUE THREAD

**Issue by yveskerwyn** [Jumpscale/lib9#83]
#### Installation information

jumpscale version: 9.2 (master)
operating system: Ubuntu 16.04

Detailed description

`j.clients.ssh.loadSSHAgent()` does load id_rsa.

Also when specifying explicitetly a keyname it doesn't work:
- `j.clients.ssh.loadSSHAgent(path="~/.ssh/id_rsa")`
- `j.clients.ssh.loadSSHAgent(path="id_rsa")`

**Comment by khaledkbadr** [comment]
fixed in https://github.com/Jumpscale/lib9/commit/cffa92b8ca8efcedde799b36d83ee86d207f74cc

**Comment by yveskerwyn** [comment]
@ khaledkbadr when testing your fix, specifically when executing `j.clients.ssh.load_ssh_key(path="~/.ssh/mykey", createkeys=True) I get this:

```
TypeError: load_ssh_key() got an unexpected keyword argument 'createkeys'
```

**Comment by yveskerwyn** [comment]
Also after executing `j.clients.ssh.start_ssh_agent()` and going back to the command line, I get the following when executing `ssh-add -l`:
```bash
Could not open a connection to your authentication agent.
```

Is this normal?

**Comment by khaledkbadr** [comment]
> TypeError: load_ssh_key() got an unexpected keyword argument 'createkeys'

should be `create_keys` as specified in the docstring

> Could not open a connection to your authentication agent. 

nope, that's not normal and I think needs investigating.

**Comment by yveskerwyn** [comment]
Thanks @khaledkbadr

I've updated the documentation about this change of argument name.

The key indeed gets created when executing `j.clients.ssh.load_ssh_key(path="~/.ssh/mykey2", create_keys=True)`, but it doesn't get loaded, which is a bug, right?

**Comment by khaledkbadr** [comment]
> but it doesn't get loaded, which is a bug, right?

If it doesn't get loaded then sure it's a bug, and needs investigating

**Comment by yveskerwyn** [comment]
Please fix and back port to 9.2.0
